ansible_collections/arista/avd/plugins/plugin_utils/schema/avdvalidator.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import absolute_import, division, print_function
+from __future__ import absolute_import, annotations, division, print_function
```

Comment:
probably no need except for annotations like for the other comment?

Reply:
removed